tests/FileServerTest.php

```diff
@@ -17,6 +20,56 @@ public function testNonExistingFileIsNotFound()
         $response = FileServer::create('non-existing-path', 'non-existing-path')
             ->getResponse();
 
-        $this->assertEquals($response->getStatusCode(), Response::HTTP_NOT_FOUND);
+        $this->assertEquals(Response::HTTP_NOT_FOUND, $response->getStatusCode(), 'Asserting that a non-existing file return a 404 status.');
+    }
+
+    /**
+     * Test to see that existing files are found
+     */
+    public function testFileIsFound()
+    {
+        $request         = $this->getMockBuilder(Request::class)->disableOriginalConstructor()->getMock();
+        $request->server = $this->getMockBuilder(ServerBag::class)->getMock();
+        $dir             = __DIR__.'/Resources';
+        $filename        = 'Foobar.txt';
+
+        $request
+            ->server
+            ->expects($this->any())
+            ->method('get')
+            ->with('REQUEST_URI')
+            ->willReturn('/Resources/'.$filename)
```

Comment:
I would like this to be a more virtual path. Like `http://host/virtual-path/real-part-of-path/real.file` should return something from `tests/Resources/real-part-of-path/real.file`.